readme.md

```diff
@@ -403,30 +403,28 @@ client.get('/demo');
 
 *Need more control over the behavior of Got? Check out the [`got.create()`](advanced-creation.md).*
 
-**Both `got.extend(options)` and `got.create(options)` will freeze the instance's default options. For `got.extend()`, the instance's default options are the result of `got.mergeOptions`, which effectively copies plain `Object` and `Array` values. Therefore, you should treat objects passed to these methods as immutable.**
```

Comment:
Why was this removed? I guess we could shorten it a little bit, but I like to keep the note about `you should treat objects passed to these methods as immutable.` somewhere.

Reply:
I consider it's no longer needed, because the defaults are cloned, so your object is safe, nothing will change :)

Reply:
👍